x-pack/legacy/plugins/lens/public/indexpattern_datasource/dimension_panel/popover_editor.tsx

```diff
@@ -188,246 +193,225 @@ export function PopoverEditor(props: PopoverEditorProps) {
   }
 
   return (
-    <EuiPopover
-      id={columnId}
-      className="lnsPopoverEditor"
-      anchorClassName={selectedColumn ? 'lnsPopoverEditor__anchor' : 'lnsPopoverEditor__link'}
-      button={
-        selectedColumn ? (
-          <EuiLink
-            className="lnsPopoverEditor__link"
-            onClick={() => {
-              setPopoverOpen(!isPopoverOpen);
+    <div id={columnId} className="lnsPopoverEditor">
+      <EuiFlexGroup gutterSize="s" direction="column">
+        <EuiFlexItem>
+          <FieldSelect
+            currentIndexPattern={currentIndexPattern}
+            existingFields={state.existingFields}
+            showEmptyFields={state.showEmptyFields}
+            fieldMap={fieldMap}
+            operationFieldSupportMatrix={operationFieldSupportMatrix}
+            selectedColumnOperationType={selectedColumn && selectedColumn.operationType}
+            selectedColumnSourceField={
+              selectedColumn && hasField(selectedColumn) ? selectedColumn.sourceField : undefined
+            }
+            incompatibleSelectedOperationType={incompatibleSelectedOperationType}
+            onDeleteColumn={() => {
+              setState(
+                deleteColumn({
+                  state,
+                  layerId,
+                  columnId,
+                })
+              );
             }}
-            data-test-subj="indexPattern-configure-dimension"
-            aria-label={i18n.translate('xpack.lens.configure.editConfig', {
-              defaultMessage: 'Edit configuration',
-            })}
-            title={i18n.translate('xpack.lens.configure.editConfig', {
-              defaultMessage: 'Edit configuration',
-            })}
-          >
-            {uniqueLabel}
-          </EuiLink>
-        ) : (
-          <>
-            <EuiButtonEmpty
-              iconType="plusInCircleFilled"
-              data-test-subj="indexPattern-configure-dimension"
-              aria-label={i18n.translate('xpack.lens.configure.addConfig', {
-                defaultMessage: 'Add a configuration',
-              })}
-              title={i18n.translate('xpack.lens.configure.addConfig', {
-                defaultMessage: 'Add a configuration',
-              })}
-              onClick={() => setPopoverOpen(!isPopoverOpen)}
-              size="xs"
-            >
-              <FormattedMessage
-                id="xpack.lens.configure.emptyConfig"
-                defaultMessage="Drop a field here"
-              />
-            </EuiButtonEmpty>
```

Comment:
For reviewers: The 'Add a configuration' button is no longer owned by the datasource.